ldap/servers/plugins/memberof/memberof.c

```diff
@@ -1020,9 +1020,16 @@ deferred_thread_func(void *arg)
      * keep running this thread until plugin is signaled to close
      */
     g_incr_active_threadcnt();
-    if (memberof_get_config()->need_fixup && perform_needed_fixup()) {
-        slapi_log_err(SLAPI_LOG_ALERT, MEMBEROF_PLUGIN_SUBSYSTEM,
-                      "Failure occured during global fixup task: memberof values are invalid\n");
+    if (memberof_get_config()->need_fixup) {
+        if (memberof_get_config()->launch_fixup) {
+            if (perform_needed_fixup()) {
+                slapi_log_err(SLAPI_LOG_ALERT, MEMBEROF_PLUGIN_SUBSYSTEM,
+                        "Failure occured during global fixup task: memberof values are invalid\n");
```

Comment:
It was there, still, - let's fix it:)
```suggestion
                        "Failure occurred during global fixup task: memberof values are invalid\n");
```